experimental/Schemes/src/Tjurina.jl

```diff
@@ -738,3 +738,88 @@ function is_contact_equivalent(X::HypersurfaceGerm, Y::HypersurfaceGerm)
   g = defining_ideal(Y)[1]
   return is_contact_equivalent(f, g)
 end
+
+
+
+
+################################################################################
+
+#####                           Tjurina module                             #####
+
+################################################################################
+
+
+@doc raw"""
+    tjurina_module(X::CompleteIntersectionGerm) 
+
+Return the Tjurina module of the complete intersection germ `(X,p)` at the point `p`.
```

Comment:
Maybe add a note that this is a special case of the T^1. I assume that T^1 and T^2 in the general case are also somewhere further behind in the pipeline?